### PR TITLE
added copy_file func

### PR DESCRIPTION
**Pull request by BGBroth**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, fname_to_copy, new_fname = command.split(" ")
```
Comment (pavlejviki) [inline app/main.py:2]: check that the splitted command consists of three parts before unpacking

File: app/main.py [hunk 2]
```diff
@@ -1 +1,11 @@
...
+
+    if not command == "cp":
+        pass
+
+    if fname_to_copy == new_fname:
+        pass
```
Comment (pavlejviki) [inline app/main.py:8]: you can join this two if conditions with `and` and better raise an informative warning instead of pass

**Review by pavlejviki — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splitted_arguments = command.split(" ")
+    if len(splitted_arguments) != 3:
+        raise ValueError("Incorrect comand provided")
+    command, fname_to_copy, new_fname = command.split(" ")
```
Comment (Dimosphen1) [inline app/main.py:5]: There is no need to perform an additional split, you could unpack the `splitted_arguments` variable.

File: app/main.py [hunk 4]
```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splitted_arguments = command.split(" ")
```
Comment (Dimosphen1) [inline app/main.py:2]: There is no need to indicate that you should split on space, it is a default option. Leave just `command.split()`.

File: app/main.py [hunk 5]
```diff
@@ -1 +1,14 @@
...
+    if len(splitted_arguments) != 3:
+        raise ValueError("Incorrect comand provided")
+    command, fname_to_copy, new_fname = command.split(" ")
+
+    if not command == "cp":
+        raise ValueError("Incorrect comand provided")
+
+    if fname_to_copy == new_fname:
+        raise ValueError("Incorrect comand provided")
```
Comment (Dimosphen1) [inline app/main.py:11]: Combine these two checks into one to reduce the number of if conditions.

File: app/main.py [hunk 6]
```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splitted_arguments = command.split(" ")
+    if len(splitted_arguments) != 3:
+        raise ValueError("Incorrect comand provided")
+    command, fname_to_copy, new_fname = command.split(" ")
+
+    if not command == "cp":
```
Comment (Dimosphen1) [inline app/main.py:7]: It would be preferable to change this line to
```suggestion
    if command != "cp":
```

File: app/main.py [hunk 7]
```diff
@@ -1 +1,14 @@
...
+        raise ValueError("Incorrect comand provided")
```
Comment (Dimosphen1) [inline app/main.py:8]: Change the typo in the `comand` word to `command`.

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested.

**Review by Dimosphen1 — APPROVED** [review]
LGTM, well done!

**Pull request closed without merging**